code/nexus/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "npm test",
-    "start": "nodemon server.js"
+    "start": "nodemon server.js",
+    "watch": "supervisor -n index.js --no-restart-on error --inspect index.js"
```

Comment:
also added `npm run watch` to the nexus™️

Refer to [this for more info](https://github.com/DanielSeehausen/fi-field-day/tree/master/code/node-server#useful-resources)

![bitmoji](https://render.bitstrips.com/v2/cpanel/4777b02c-71dc-4081-9c03-26d7740bf7c5-7c243825-53cb-4c20-a7d9-3fc8d332e86d-v1.png?transparent=1&palette=1&width=246)